instruqt-tracks/terraform-cloud-aws-v2/track_scripts/setup-workstation

```diff
@@ -0,0 +1,191 @@
+#!/bin/bash -l
+# Track setup script
+
+set -e
+
+# Wait for Instruqt to finish booting the VM
+# This is better than doing sleep
+while [ ! -f /opt/instruqt/bootstrap/host-bootstrap-completed ]
+do
+    echo "Waiting for Instruqt to finish booting the VM"
+    sleep 1
+done
+
+# To repurpose this script, we choose the target 
+# Cloud Environment using this variable
+# export CLOUD_ENV="aws"
+
+# We use GitHub gh to load secrets in our working repo
+# For everything else, we use git
+snap install gh
+
+# Ensure we're working on the home directory
+cd /root
+
+# Get our working assets
+echo "Cloning assets..."
+git clone https://github.com/hashicorp/se-workshop-hashicat-app.git
+echo "Done cloning assets..."
+
+mv /root/se-workshop-hashicat-app/hashicat-$CLOUD_ENV /root/hashicat-$CLOUD_ENV
+mv /root/se-workshop-hashicat-app/hashicat-policies /root/hashicat-$CLOUD_ENV/policies
+mv /root/hashicat-$CLOUD_ENV/policies/$CLOUD_ENV/* /root/hashicat-$CLOUD_ENV/policies/.
+
+rm -fR  /root/hashicat-$CLOUD_ENV/policies/$CLOUD_ENV
+rm -fR  /root/hashicat-$CLOUD_ENV/policies/azure
+rm -fR  /root/hashicat-$CLOUD_ENV/policies/gcp
+
+mv /root/se-workshop-hashicat-app/terraform-cloud /root/terraform-cloud
+mv /root/se-workshop-hashicat-app/terraform-github /root/terraform-github
+mv /root/se-workshop-hashicat-app/terraform-api /root/terraform-api
+mv /root/se-workshop-hashicat-app/scripts /root/scripts
+rm -fR /root/se-workshop-hashicat-app
+
+# We should end up with the following structure:
+#
+#  ├── hashicat-aws
+#  │   ├── files
+#  │   └── policies
+#  ├── terraform-cloud
+#  │   └── exercises
+#  └── terraform-github
+#  └── terraform-api
+#  └── scripts
+
+# Set the initial workspace name
+export WORKSPACE="hashicat-${CLOUD_ENV}"
+cp /root/terraform-cloud/terraform.tfvars.example /root/terraform-cloud/terraform.tfvars
+
+cd /root/terraform-cloud
+sed -i "s/YOUR_WORKSPACE/$WORKSPACE/g" terraform.tfvars
+
+# We do this to ear-mark the variables associated 
+# with the environment in the TFC workspace
+sed -i "s/CLOUD_ENV/$CLOUD_ENV/g" variables.tf
+
+# Our only commit/pull directory
+GITDIR="/root/hashicat-$CLOUD_ENV"
```

Comment:
There's a `GIT_DIR` environment variable, which this is close to. What's this used for? I don't see any `git` commands in the rest of this file, and we don't export this or put it in a shell startup file for consumption later.

Reply:
That's legacy. Can remove